app/assets/stylesheets/application.css.scss

```diff
@@ -11,6 +11,13 @@
  * file per style scope.
  *
  * 
+ *= require 'masonry/basic'
+ *= require 'masonry/centered'
+ *= require 'masonry/fluid'
+ *= require 'masonry/gutters'
+ *= require 'masonry/infinitescroll'
+ *= require 'masonry/right-to-left'
```

Comment:
you a loading a lot of files here (same with application.js), now that it's working you should check which ones are really necessary